src/quadratic_expression.cpp

```diff
@@ -32,8 +32,8 @@ Rcpp::List quadratic_expression(Eigen::Map<Eigen::VectorXd> X,
     // Auto differential.
     auto f = ad::autodiff(expr, seed.array());
 
-    Rcpp::List res = Rcpp::List::create(Rcpp::Named("value") = f,
-                                        Rcpp::Named("gradient") = x.get_adj());
+    Rcpp::List res = Rcpp::List::create(Rcpp::Named("value") = f(0,0),
```

Comment:
This can be avoided if you remove lines 30-31 and not pass the second argument to `ad::autodiff`.

Reply:
Nice. Will try.

Reply:
Sorry I get `ad::autodiff(expr)` rejected for need ing two args.  Which we don't in other places.  I am confused.